plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -167,6 +168,7 @@ def get_token(module_params):
     return {
         'Authorization': 'Bearer ' + token,
         'Content-Type': 'application/json'
+        'User-Agent': headers['User-Agent']
```

Comment:
```suggestion
        'Content-Type': 'application/json',
        'User-Agent': headers['User-Agent'],
```

Reply:
removed the explicit header and used http_agent param